### DISCUSSION TITLE
Supabase Database table unable to add more column even if it is empty

### DISCUSSION THREAD

**Discussion by pushkarsingh32** [supabase/supabase#17494] [Questions]
![supabase showing error on adding column](https://github.com/supabase/supabase/assets/29558481/740824aa-a0d7-4318-bde0-229e8f7bafc4)

Is there any way I can overcome this ? I need to add more columns in this table but Supabase is throwing error

**Comment by GaryAustin1** [comment]
https://github.com/orgs/supabase/discussions/17296
There is at least one browser extension that by default blocks PATCH for some reason.
